suite-native/atoms/src/Sheet/BottomSheetFlashList.tsx

```diff
@@ -0,0 +1,146 @@
+import { useEffect, useRef, ReactNode } from 'react';
+import { useSafeAreaInsets } from 'react-native-safe-area-context';
+import Animated from 'react-native-reanimated';
+import { PanGestureHandler } from 'react-native-gesture-handler';
+import { GestureResponderEvent, Pressable } from 'react-native';
+
+import { FlashList, FlashListProps } from '@shopify/flash-list';
+
+import { prepareNativeStyle, useNativeStyles } from '@trezor/styles';
+
+import { Box } from '../Box';
+import { BottomSheetContainer } from './BottomSheetContainer';
+import { useBottomSheetAnimation } from './useBottomSheetAnimation';
+import { BottomSheetHeader } from './BottomSheetHeader';
+
+export type BottomSheetFlashListProps<TItem> = {
+    isVisible: boolean;
+    isCloseDisplayed?: boolean;
+    onClose: (isVisible: boolean) => void;
```

Comment:
Do we wanna keep mandatory properties together at the beginning?

Reply:
I think overall consensus is that we want keep them in begging but I think it doesn't matter. In case of these code "stylystic" issue it should either have autofixable eslint rule or shoudn't be enforced.

Reply:
Why do you think it's enforced? I'm just trying to figure out what's the team consensus.